Source/DafnyTestGeneration/DeadCodeCommand.cs

```diff
@@ -21,6 +27,8 @@ public class DeadCodeCommand : ICommandSpec {
   }
 
   public void PostProcess(DafnyOptions dafnyOptions, Options options, InvocationContext context) {
+    // IMPORTANT: Before adding new default options, make sure they are
```

Comment:
This seems frail. Why not make the code more robust instead of adding this comment? Why doesn't CopyForProcedure use the DafnyOptions clone constructor?